### PR TITLE
130 feature custom html error page integration on 403 like captcha but for ban

### PR DESCRIPTION
**Pull request by mathieuHa**

### CODE REVIEW

File: ban.html [hunk 1]
```diff
@@ -326,4 +326,5 @@ <h1 class="text-2xl lg:text-3xl xl:text-4xl">CrowdSec Access Forbidden</h1>
     </div>
   </div>
 </body>
-</html>
+
+</html>
```
Comment (maxlerebourg) [inline ban.html:330]: nop

File: pkg/configuration/configuration.go [hunk 2]
```diff
@@ -67,8 +67,8 @@ type Config struct {
 	RedisCachePassword                       string   `json:"redisCachePassword,omitempty"`
 	RedisCachePasswordFile                   string   `json:"redisCachePasswordFile,omitempty"`
 	RedisCacheDatabase                       string   `json:"redisCacheDatabase,omitempty"`
-	BanHTMLFilePath                          string   `json:"banHtmlFilePath,omitempty"`
 	CaptchaHTMLFilePath                      string   `json:"captchaHtmlFilePath,omitempty"`
+	BanHTMLFilePath                          string   `json:"banHtmlFilePath,omitempty"`
```
Comment (maxlerebourg) [inline pkg/configuration/configuration.go:71]: Why this change ?

**Review by maxlerebourg — APPROVED** [review]

File: pkg/configuration/configuration.go (hunk 2 above)
Comment (mathieuHa) [reply pkg/configuration/configuration.go]: Idea was to not use template for static
